core/templates/base-components/base-content.component.html

```diff
@@ -45,10 +45,9 @@
                 <div class="oppia-base-content-pre-logo-container">
                   <ng-content select="navbar-pre-logo-action"></ng-content>
                 </div>
-                <top-navigation-bar [headerText]="getHeaderText"
-                                    [subheaderText]="getSubheaderText"
-                                    [toggle]="toggleSidebar.bind(this)">
-                </top-navigation-bar>
+                  <oppia-top-navigation-bar [headerText]="getHeaderText"
+                                            [subheaderText]="getSubheaderText">
```

Comment:
call these functions right here and pass the values instead of callbacks.

Reply:
done